### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by vakhotsky**

### CODE REVIEW

File: src/getHumanAge.test.js [hunk 1]
```diff
@@ -7,4 +7,40 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+  test('return array = 0 if function is empty', () => {
+    expect(getHumanAge(0, 0))
+      .toEqual([0, 0]);
+  });
```
Comment (YevhS) [inline src/getHumanAge.test.js:13]: You should have separation between tests, add empty line between tests

File: src/getHumanAge.test.js [hunk 2]
```diff
@@ -7,4 +7,40 @@ describe('getHumanAge', () => {
...
+  test('First 15 **cat** years give 1 **human** year', () => {
```
Comment (YevhS) [inline src/getHumanAge.test.js:14]: Why do you use asterisks in the names of test? What about `""`?

File: src/getHumanAge.test.js [hunk 3]
```diff
@@ -7,4 +7,40 @@ describe('getHumanAge', () => {
...
+    expect(getHumanAge(0, 15))
+      .toEqual([0, 1])
+  });
+  test('the next 9 **cat** years give 1 more **human** year', () => {
```
Comment (YevhS) [inline src/getHumanAge.test.js:18]: Names of tests shouldn't be a continuation of a previous test. Every test should be independent, and the name of the test as well. You can use structure `... should be returned for the input ..` for example.

File: src/getHumanAge.test.js [hunk 4]
```diff
@@ -7,4 +7,40 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+  test('return array = 0 if function is empty', () => {
```
Comment (YevhS) [inline src/getHumanAge.test.js:10]: `function is empty` is not a correct assertion, when parameters are 0, 0 maybe?

**Review by YevhS — CHANGES_REQUESTED** [review]
You should add tests to check boundary values
